lib/cart/cart_repository.dart

```diff
@@ -101,4 +104,32 @@ class CartRepository {
       return false;
     }
   }
+
+  Future<void> changeQuantityInCart({
+    required int entryNumber,
+    required int quantity,
+  }) async {
+    const cartCode = kTestCart;
+
+    try {
+      await _cartApi.changeQuantityInCart(
+        cartCode: cartCode,
+        entryNumber: entryNumber,
+        quantity: quantity,
+      );
+
+      // TODO: add any applicable cart messages
+      return;
+    } catch (e) {
+      _cartMessageStreamController.add(
+        AddToCartMessage(
+          CartMessageType.error,
+          'Failed to change quantity of $entryNumber to $quantity',
+        ),
+      );
+    } finally {
+      final cart = await _cartApi.fetchCart(cartCode: kTestCart);
```

Comment:
I wouldn't refresh the cart in the `finally` clause, if something went wrong, keep the state wrong so the dev has a chance to understand what happened